### DISCUSSION TITLE
How to only fetch and read data from the database in flutter?

### DISCUSSION THREAD

**Discussion by abbasyusuf** [supabase/supabase#5428] [Questions]
I just want to fetch and display the data in a list in one simple page in flutter, nothing more, how can I do that? 
I have 2 column in my table: Id, channel, description. 

Any help?

**Comment by dshukertjr** [comment]
Hi @abbasyusuf,

You can use [`select()`](https://supabase.com/docs/reference/dart/select) statement to retrieve data from your table. You can then use a [`FutureBuilder`](https://api.flutter.dev/flutter/widgets/FutureBuilder-class.html) to display the retrieved data!
